packages/theme/components/FormComponents/EmailInput.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <SfInput
+      @click="e => $emit('click', e)"
+      @input="e => $emit('input', e)"
+      v-bind="{...$attrs, ...$props}"
+      :class="{'disable-input': disabled}"
+      :valid="!errors[0]"
+      :errorMessage="errors[0]"
+    />
+  </div>
+</template>
+
+<script>
+import {
+  SfInput
+} from '@storefront-ui/vue';
+
+export default {
+  name: 'EmailInput',
+  props: {
+    disabled: {
+      type: Boolean,
+      default() {
+        return false;
+      }
+    },
+    label: {
```

Comment:
The `SfInput` label is empty by default, our prop should be removed.
https://docs.storefrontui.io/?path=/docs/components-atoms-input--common